sycl/source/detail/scheduler/commands.cpp

```diff
@@ -1617,6 +1620,30 @@ static void ReverseRangeDimensionsForKernel(NDRDescT &NDR) {
   }
 }
 
+// Creates an image1d buffer wrapper object to enable surface index-based
+// access for ESIMD.
+static pi_mem createImage1dBufferWrapper(const context_impl &Ctx, size_t Size,
```

Comment:
Conceptually this function should be in program_manager.cpp. Could you please move it there?

Reply:
do you mean memory manager?

Reply:
Yes, I meant memory manager